dist/menu-button/menu-button.css

```diff
@@ -1,18 +1,18 @@
+:root {
+  --bubble-filter: drop-shadow(0 2px 7px rgba(0, 0, 0, 0.15)) drop-shadow(0 5px 17px rgba(0, 0, 0, 0.2));
```

Comment:
Since this is used in multiple places  we should only have it in a global place. Like maybe in the mixin file?
That way it won't be defined for each file.

Reply:
So...this is a precompiled `css` file. The var already IS defined in a mixin, `bubble-mixin.less`. The problem is that when it was defined, it was defined as a `css` prop in a `:root` and this is why it proliferates into all the `css` files from all the `less` files that use the mixin. 

I think this is out of scope for this specific issue. I'm merely employing the same strategy that has been in use. It seems like if we want to fix this, it should be a separate issue that fixes it for all instances/uses.

Reply:
Got it, yes you are right.